config_defaults/subtests/docker_cli/negativeusage.ini

```diff
@@ -70,24 +70,6 @@ subarg = -e,PATH=/tmp,%%(FQIN)s,true
 stderr = exec\: \"true\"\: executable file not found in \$PATH
 extcmd = 127
 
-[docker_cli/negativeusage/iv5]
-subcmd = run
-subarg = --tmpfs,/dev,%%(FQIN)s
-stderr =  no such file or directory
-extcmd = 127
-
-[docker_cli/negativeusage/iv6]
-subcmd = run
-subarg = --tmpfs,:rw,%%(FQIN)s
-stderr =  open /proc/self/task/1/attr/exec: no such file or directory
-extcmd = 125
-
-[docker_cli/negativeusage/iv7]
-subcmd = run
-subarg = --tmpfs,.,%%(FQIN)s
-stderr =  Message: Failed to open /dev/null - open /dev/null: permission denied
-extcmd = 125
```

Comment:
Unfortunately, the test still fails. docker-1.10 exits with 125, 1.12 with 126.

I would really like to get rid of all failing tests. I really want to get ADEPT running cleanly, so when it fails I can actually look at the failure instead of shrugging and saying "oh, it always fails". If and when there is a defined behavior for `--tmpfs .`, I'd like to add a test for it. But right now I really just want to get tests passing.